roles/ceph-infra/tasks/setup_ntp.yml

```diff
@@ -10,9 +10,30 @@
         ntp_service_name: ntpd
       when: ansible_os_family in ['RedHat', 'Suse']
 
-- name: setup ntp daemon
+# Installation of NTP daemons needs to be a separate task since installations
+# can't happen on Atomic
+- name: install the ntp daemon
+  when: not is_atomic
   block:
-    - name: install and enable timesyncd
+    - name: install ntpd
+      package:
+        name: ntp
+        state: present
+      register: result
+      until: result is succeeded
+      when: ntp_daemon_type == "ntpd"
```

Comment:
keep consistency with the rest of the playbook.


change from:
`when: ntp_daemon_type == "ntpd"`

to:
```
when:
  - ntp_daemon_type == "ntpd"
```

Reply:
I find this way of writing conditions very odd. But I admit that's only because I am used to writing C/C++.

Reply:
actually it could make sense to have everything in one line when only 1 condition is present and everything split on multiple lines when more than 1 condition but all conditions in the playbook are written this way whatever there is 1 or several conditions, so to keep consistency, let's write all condition the same way it has been done until now.

I would add that sometimes we must adapt to the project policies even if we personally find something is very odd. (Imagine if everyone could do things the way they like...)

Reply:
> actually it could make sense to have everything in one line when only 1 condition is present and everything split on multiple lines when more than 1 condition but all conditions in the playbook are written this way whatever there is 1 or several conditions, so to keep consistency, let's write all condition the same way it has been done until now.
> 
> I would add that sometimes we must adapt to the project policies even if we personally find something is very odd. (Imagine if everyone could do things the way they like...)

I totally agree with this.

I hope I am not annoying you with my coding styles. I guess I am.  :P